plugins/connectrpc/es/v1.1.4/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# syntax=docker/dockerfile:1.6
+FROM node:20.10.0-alpine3.18 AS build
+WORKDIR /app
+COPY --link package*.json .
+RUN npm ci
+RUN sed -i -e 's|/usr/bin/env node|/nodejs/bin/node|g' /app/node_modules/@connectrpc/protoc-gen-connect-es/bin/protoc-gen-connect-es
+
+FROM gcr.io/distroless/nodejs20-debian12:latest@sha256:6bd06494907b362c5d85dfbf60930e98272b9ffc1cbcb5c6d602da7ba9a303a0
```

Comment:
I miss seeing versions here, but I trust in the process now.

Reply:
For the distroless ones, we didn't have versions before (we even left off the tag in some cases).